docs/standard/serialization/binary-serialization.md

```diff
@@ -85,10 +131,42 @@ As the nature of binary serialization allows the modification of private members
 - <xref:System.Data.SqlTypes.SqlInt16?displayProperty=nameWithType>   
 - <xref:System.Data.SqlTypes.SqlInt32?displayProperty=nameWithType>   
 - <xref:System.Data.SqlTypes.SqlInt64?displayProperty=nameWithType>   
+- <xref:System.Data.SqlTypes.SqlNotFilledException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Data.SqlTypes.SqlNullValueException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
 - <xref:System.Data.SqlTypes.SqlString?displayProperty=nameWithType>   
+- <xref:System.Data.SqlTypes.SqlTruncateException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Data.SqlTypes.SqlTypeException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Data.StrongTypingException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Data.SyntaxErrorException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Data.VersionNotFoundException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.DataMisalignedException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
 - <xref:System.DateTime?displayProperty=nameWithType>   
 - <xref:System.DateTimeOffset?displayProperty=nameWithType>   
 - <xref:System.Decimal?displayProperty=nameWithType>   
+- <xref:System.Diagnostics.Contract.ContractException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Diagnostics.Tracing.EventSourceException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.DirectoryNotFoundException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
```

Comment:
This should be \<xref:System.IO.DirectoryNotFoundException?displayProperty=nameWithType>